### PR TITLE
Web development - modified Challenges segment.

### PR DESCRIPTION
**Pull request by Braj-01**
**IMPORTANT:** When creating your Pull Request, please use the following format for the title: `Category - Resource Name & Section`  

PR Title samples:

- `Web development - CSS Portal & Tools`
- `Web development - Leetcode & Challenges`

### Checklist:
Before submitting your pull request, ensure that you have completed the following tasks: 

- [x] I have carefully reviewed and adhered to the [contributing guidelines](https://github.com/jfmartinz/ResourceHub/blob/main/CONTRIBUTING.md) before creating this pull request.
- [x] I followed the prescribed PR title template. _(Check this if you're adding a resource)_

### CODE REVIEW

**Comment by Braj-01** [comment]
Pls see second pull request because in first there was some error.

File: Web Development/README.md [hunk 1]
```diff
@@ -1418,10 +1418,25 @@ This section contains hundreds of coding challenges that test your knowledge of
         <th>Resource Name</th>
         <th>Description</th>
       </tr>
+      <tr>
+      <td><a href="https://www.hackerrank.com/interview/preparation-kits/">Problem Solving</a></td>
```
Comment (jfmartinz) [inline Web Development/README.md:1422]:
```suggestion
      <td><a href="https://www.hackerrank.com/interview/preparation-kits/">HackerRank Preparation Kits</a></td>
```

**Review by jfmartinz — CHANGES_REQUESTED** [review]

File: Web Development/README.md
Comment (Braj-01) [inline Web Development/README.md:1]: This is the changes that I have made.

**Review by jfmartinz — APPROVED** [review]

**Comment by jfmartinz** [comment]
Thanks

**Comment by Braj-01** [comment]
You're welcome!

**Pull request merged**